tests/integration/questionnaires.test.js

```diff
@@ -245,6 +257,20 @@ describe('QUESTIONNAIRES ROUTES - PUT /questionnaires/{_id}', () => {
       expect(response.statusCode).toBe(200);
     });
 
+    it('should update cards status', async () => {
```

Comment:
Est-ce qu'il est/sera possible de dé-publier un questionnaire ?

Reply:
D'ailleurs ici, c'est plutôt : `'should update questionnaire status'`, non ?

Reply:
archiver, dépublier je ne pense pas